ospf6d/ospf6_asbr.c

```diff
@@ -1305,9 +1361,14 @@ DEFUN (ospf6_redistribute_routemap,
 	if (type < 0)
 		return CMD_WARNING_CONFIG_FAILED;
 
-	ospf6_asbr_redistribute_unset(type, ospf6->vrf_id);
-	ospf6_asbr_routemap_set(type, argv[idx_word]->arg, ospf6->vrf_id);
+	red = ospf6_redist_add(ospf6, type, 0);
```

Comment:
Here the type is defined as uint8 but above it is defined as int, and then in unset function again it is defined as int, can you keep it same in all the places?

Reply:
Sure, added